tests/saitests/sai_qos_tests.py

```diff
@@ -1326,11 +1325,6 @@ def runTest(self):
             for cntr in self.ingress_counters:
                 assert(recv_counters[cntr] > recv_counters_bases[sidx_dscp_pg_tuples[i][0]][cntr])
 
-            # assert no egress drop at the dut xmit port
```

Comment:
This check is important and removing it, we may miss actual issues. During the test flow, it's hard to say if this counter was increased due to test packets or other packets. This test runs on all platforms that has headroom pool so why is this issue happening only on this particular setup? Besides, why are arp/ndp packets getting dropped?

Reply:
Neetha, just revert previous changes and re-order the code sequence to avoid the flaky issue.